CHANGELOG

```diff
@@ -1,3 +1,53 @@
+4.0.0.pre3 (191 commits)
+----------------
+
+New features (1)
+----------------
+
+* In pre2 we updated our gem layout and found a few things we didn't like.
+  pre3 is now shaped how we expect to hold through 4.0!
+
+  * shoes -- Now a meta-gem with the default installation as dependencies
```

Comment:
"and no code of its own" maybe, dunno if everyone knows what a meta gem or what we use the term meta gem for :-)